ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  ansible-operator:
+    name: "4.9"
+    namespace: ocp
+    tag: ansible-operator
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: build/custom-ci-Dockerfile
+  use_build_cache: true
+images:
+- dockerfile_path: build/Dockerfile
+  from: ansible-operator
+  to: oadp-operator
+operator:
+  bundles:
+  - dockerfile_path: build/Dockerfile.bundle
+  substitutions:
+  - pullspec: quay.io/konveyor/oadp-operator.v0.2.3:latest
+    with: oadp-operator
+promotion:
```

Comment:
Do we want to promote the images? is someone going to use this?

Reply:
Removing Promote causes error. I have tried remove twice. I failed ci/prow/config with error "run make jobs" to add promotion.

Reply:
Removing `promotion` is completely fine. The check correclty instructed you to run `make jobs` (which you should do) simply because not having `promotion` makes generated jobs different.